[android] Reworked descriptions in the editor (#6173)
organicmaps/organicmaps: data/strings/strings.txt

  [editor_other_info]
    tags = android,ios

>     en = Open a discussion note

Reviewer: ?
```
    en = Open a discussion note at OpenStreetMap.org
```

Author: I think with the description below `OpenStreetMap.org` would be rather redundant

  [editor_detailed_description]
    tags = android,ios

>     en = Your edits are uploaded to the public OpenStreetMap (OSM) database, so don't add personal or copyrighted information.

Reviewer: What do you think about this?

    en = Your edits are uploaded to the public OpenStreetMap.org (OSM) database. Please do not add personal or copyrighted information.

Reviewer 2: I think we can remove public in sentence

Reviewer: No, it's critically important, based on our experience.

Author: I check all OM edits with a boundary box intersecting munich and there are regularly people who upload personal notes

Author: I updated the description

  [editor_other_info]
    tags = android,ios

>     en = Note to OpenStreetMap volunteers

Reviewer: en = Note to OpenStreetMap volunteers
    af = Nota aan OpenStreetMap-vrywilligers
    ar = ملاحظة لمتطوعي OpenStreetMap
    az = OpenStreetMap könüllüləri üçün qeyd
    be = Заўвага для валанцёраў OpenStreetMap
    bg = Забележка към доброволците на OpenStreetMap
    ca = Nota per als voluntaris d'OpenStreetMap
    cs = Poznámka pro dobrovolníky OpenStreetMap
    da = Note til OpenStreetMap-frivillige
    de = Hinweis für OpenStreetMap-Freiwillige
    el = Σημείωση για τους εθελοντές του OpenStreetMap
    es = Nota para los voluntarios de OpenStreetMap
    et = Märkus OpenStreetMapi vabatahtlikele
    eu = Oharra OpenStreetMap-eko boluntarioentzat
    fa = به داوطلبان OpenStreetMap توجه داشته باشید
    fi = Huomautus OpenStreetMapin vapaaehtoisille
    fr = Note aux bénévoles d'OpenStreetMap
    he = הערה למתנדבי OpenStreetMap
    hi = OpenStreetMap स्वयंसेवकों के लिए नोट
    hu = Megjegyzés az OpenStreetMap önkénteseinek
    id = Catatan untuk relawan OpenStreetMap
    it = Nota per i volontari di OpenStreetMap
    ja = OpenStreetMapボランティアへの注意
    ko = 오픈스트리트맵 자원 봉사자 참고 사항
    lt = Pastaba "OpenStreetMap" savanoriams
    mr = OpenStreetMap स्वयंसेवकांसाठी टीप
    nb = Merknad til OpenStreetMap-frivillige
    nl = Opmerking voor OpenStreetMap vrijwilligers
    pl = Uwaga dla wolontariuszy OpenStreetMap
    pt = Nota para os voluntários do OpenStreetMap
    ro = Notă pentru voluntarii OpenStreetMap
    ru = Примечание для волонтеров OpenStreetMap
    sk = Poznámka pre dobrovoľníkov OpenStreetMap
    sv = Meddelande till OpenStreetMap-volontärer
    sw = Kumbuka kwa wanaojitolea wa OpenStreetMap
    th = หมายเหตุถึงอาสาสมัคร OpenStreetMap
    tr = OpenStreetMap gönüllülerine not
    uk = До уваги волонтерів OpenStreetMap
    vi = Lưu ý dành cho tình nguyện viên OpenStreetMap
    zh-Hans = OpenStreetMap 志愿者须知
    zh-Hant = OpenStreetMap 志工注意事項

Reviewer: Let's check how it looks with new strings (also check the longest translations language).
```
    en = Note to OpenStreetMap volunteers (optional)
    af = Nota aan OpenStreetMap-vrywilligers (opsioneel)
    ar = ملاحظة لمتطوعي OpenStreetMap (اختياري)
    az = OpenStreetMap könüllüləri üçün qeyd (isteğe bağlı)
    be = Заўвага для валанцёраў OpenStreetMap (неабавязкова)
    bg = Забележка към доброволците на OpenStreetMap (по избор)
    ca = Nota per als voluntaris d'OpenStreetMap (opcional)
    cs = Poznámka pro dobrovolníky OpenStreetMap (nepovinné)
    da = Note til OpenStreetMap-frivillige (valgfrit)
    de = Hinweis für OpenStreetMap-Freiwillige (optional)
    el = Σημείωση για τους εθελοντές του OpenStreetMap (προαιρετική)
    es = Nota para los voluntarios de OpenStreetMap (opcional)
    et = Märkus OpenStreetMapi vabatahtlikele (vabatahtlik)
    eu = OpenStreetMap-eko boluntarioentzako oharra (aukerakoa)
    fa = توجه به داوطلبان OpenStreetMap (اختیاری)
    fi = Huomautus OpenStreetMapin vapaaehtoisille (valinnainen)
    fr = Note aux volontaires OpenStreetMap (facultatif)
    he = הערה למתנדבי OpenStreetMap (אופציונלי)
    hi = OpenStreetMap स्वयंसेवकों के लिए नोट (वैकल्पिक)
    hu = Megjegyzés az OpenStreetMap önkénteseinek (nem kötelező)
    id = Catatan untuk relawan OpenStreetMap (opsional)
    it = Nota per i volontari di OpenStreetMap (opzionale)
    ja = OpenStreetMapボランティアへの注意（オプション）
    ko = 오픈스트리트맵 자원 봉사자 참고 사항(선택 사항)
    lt = Pastaba OpenStreetMap savanoriams (neprivaloma)
    mr = OpenStreetMap स्वयंसेवकांसाठी टीप (पर्यायी)
    nb = Merknad til OpenStreetMap-frivillige (valgfritt)
    nl = Opmerking voor OpenStreetMap vrijwilligers (optioneel)
    pl = Uwaga dla wolontariuszy OpenStreetMap (opcjonalnie)
    pt = Nota para os voluntários do OpenStreetMap (opcional)
    ro = Notă pentru voluntarii OpenStreetMap (opțional)
    ru = Примечание для волонтеров OpenStreetMap (необязательно)
    sk = Poznámka pre dobrovoľníkov OpenStreetMap (nepovinné)
    sv = Meddelande till OpenStreetMap-volontärer (valfritt)
    sw = Kumbuka kwa wanaojitolea wa OpenStreetMap (si lazima)
    th = หมายเหตุถึงอาสาสมัคร OpenStreetMap (ไม่บังคับ)
    tr = OpenStreetMap gönüllülerine not (isteğe bağlı)
    uk = Примітка для волонтерів OpenStreetMap (необов'язково)
    vi = Lưu ý dành cho tình nguyện viên OpenStreetMap (tùy chọn)
    zh-Hans = OpenStreetMap 志愿者须知（可选）
    zh-Hant = OpenStreetMap 志工注意事項（可選）
```

Author: The longest string, stretching over two lines, works as well:
![Screenshot_2024-08-24-20-48-46-520_app organicmaps debug](https://github.com/user-attachments/assets/7f53e796-d528-457c-8f23-9ea807a70dc1)

  [editor_other_info]
    tags = android,ios

>     en = Note to OpenStreetMap volunteers (optional)

Reviewer: @map-per adding `comment = ` with a bit more context would also be helpful.

@organicmaps/translations please help with reviewing these translations. Here is the image with the example of their use:

![выява](https://github.com/user-attachments/assets/c999f6a8-fd9a-4e5e-aa97-71e949d71011)

    bg = Описване на грешки в картата или неща, които не могат да бъдат редактирани с Organic Maps
    ca = Descriu errors al mapa o què no es pot editar amb Mapes orgànics
    cs = Popište chyby na mapě nebo věci, které nelze upravit pomocí aplikace Organic Maps.
    da = Beskriv fejl på kortet eller ting, der ikke kan redigeres med Organic Maps
    de = Beschreiben Sie Kartenfehler oder Dinge, die mit Organic Maps nicht bearbeitet werden können
    el = Περιγράψτε λάθη στο χάρτη ή πράγματα που δεν μπορούν να επεξεργαστούν με τους Organic Maps
    es = Describa los errores en el mapa o las cosas que no se pueden editar con Organic Maps
    et = Kirjeldage kaardil olevaid vigu või asju, mida ei saa Organic Mapsiga muuta.
    eu = Deskribatu mapan akatsak edo mapa organikoak erabiliz editatu ezin dena
    fa = خطاهای موجود در نقشه یا مواردی که با استفاده از نقشه های ارگانیک قابل ویرایش نیستند را شرح دهید
    fi = Kuvaile kartassa olevia virheitä tai asioita, joita ei voi muokata Organic Mapsilla.
    fr = Décrivez les erreurs sur la carte ou les éléments qui ne peuvent pas être modifiés avec Organic Maps.

>     he = תאר שגיאות במפה או מה שלא ניתן לערוך באמצעות מפות אורגניות

Reviewer 3: ```suggestion
    he = תארו שגיאות במפה או דברים שלא ניתן לערוך באמצעות Organic Maps
```

Other auto-translations are good

    mr = तुमची संपादने सार्वजनिक <a href="https://wiki.openstreetmap.org/wiki/About_OpenStreetMap">OpenStreetMap</a> डेटाबेसवर अपलोड केली जातात. कृपया वैयक्तिक किंवा कॉपीराइट केलेली माहिती जोडू नका.
    nb = Dine endringer lastes opp til den offentlige <a href="https://wiki.openstreetmap.org/wiki/About_OpenStreetMap">OpenStreetMap</a>-databasen. Vennligst ikke legg til personlig eller opphavsrettsbeskyttet informasjon.
    nl = Uw bewerkingen worden geüpload naar de openbare <a href="https://wiki.openstreetmap.org/wiki/NL:Wat_is_OpenStreetMap%3F">OpenStreetMap</a> database. Voeg geen persoonlijke of auteursrechtelijk beschermde informatie toe.
    pl = Państwa zmiany są przesyłane do publicznej bazy danych <a href="https://wiki.openstreetmap.org/wiki/Pl:Wstęp">OpenStreetMap</a>. Prosimy nie dodawać informacji osobistych lub chronionych prawem autorskim.
    pt = As suas edições são carregadas para a base de dados pública <a href="https://wiki.openstreetmap.org/wiki/Pt:Sobre_o_OpenStreetMap">OpenStreetMap</a>. Por favor, não adicione informações pessoais ou protegidas por direitos de autor.

>     pt-BR = Suas edições são carregadas no banco de dados público <a href="https://wiki.openstreetmap.org/wiki/Pt:Sobre_o_OpenStreetMap">OpenStreetMap</a>. Não adicione informações pessoais ou protegidas por direitos autorais.

Reviewer 4: ```suggestion
    pt-BR = As suas edições são enviadas à base de dados pública <a href="https://wiki.openstreetmap.org/wiki/Pt:Sobre_o_OpenStreetMap">OpenStreetMap</a>. Não adicione informações pessoais ou protegidas por direitos autorais.
```

    zh-Hans = 给 OSM 社区发送标记
    zh-Hant = 給 OSM 社區發送標記

  [editor_note_hint]
    tags = android

>     en = Describe map errors or additional details that cannot be edited with Organic Maps (optional)

Reviewer: 1. Maybe remove `(optional)` from here and write `Optional note to OpenStreetMap volunteers` in the title?
2. What about a shorter version?
```
    en = Describe errors on the map or things that cannot be edited with Organic Maps
    af = Beskryf foute in die kaart of wat nie met Organiese Kaarte geredigeer kan word nie
    ar = وصف الأخطاء على الخريطة أو الأشياء التي لا يمكن تحريرها باستخدام الخرائط العضوية
    az = Xəritədəki səhvləri və ya Üzvi Xəritələrdən istifadə edərək redaktə edilə bilməyənləri təsvir edin
    be = Апішыце памылкі на карце ці тое, што нельга рэдагаваць з дапамогай Organic Maps
    bg = Описване на грешки в картата или неща, които не могат да бъдат редактирани с Organic Maps
    ca = Descriu errors al mapa o què no es pot editar amb Mapes orgànics
    cs = Popište chyby na mapě nebo věci, které nelze upravit pomocí aplikace Organic Maps.
    da = Beskriv fejl på kortet eller ting, der ikke kan redigeres med Organic Maps
    de = Beschreiben Sie Fehler auf der Karte oder Dinge, die nicht mit Organic Maps bearbeitet werden können
    el = Περιγράψτε λάθη στο χάρτη ή πράγματα που δεν μπορούν να επεξεργαστούν με τους Organic Maps
    es = Describa los errores en el mapa o las cosas que no se pueden editar con Organic Maps
    et = Kirjeldage kaardil olevaid vigu või asju, mida ei saa Organic Mapsiga muuta.
    eu = Deskribatu mapan akatsak edo mapa organikoak erabiliz editatu ezin dena
    fa = خطاهای موجود در نقشه یا مواردی که با استفاده از نقشه های ارگانیک قابل ویرایش نیستند را شرح دهید
    fi = Kuvaile kartassa olevia virheitä tai asioita, joita ei voi muokata Organic Mapsilla.
    fr = Décrivez les erreurs sur la carte ou les éléments qui ne peuvent pas être modifiés avec Organic Maps.
    he = תאר שגיאות במפה או מה שלא ניתן לערוך באמצעות מפות אורגניות
    hi = मानचित्र में त्रुटियों का वर्णन करें या ऑर्गेनिक मानचित्र का उपयोग करके क्या संपादित नहीं किया जा सकता है
    hu = Írja le a térképen található hibákat vagy olyan dolgokat, amelyeket nem lehet az Organic Maps segítségével szerkeszteni.
    id = Jelaskan kesalahan pada peta atau hal-hal yang tidak dapat diedit dengan Peta Organik
    it = Descriva gli errori sulla mappa o le cose che non possono essere modificate con Organic Maps
    ja = 地図上のエラーやOrganic Mapsで編集できないものについて説明します。
    ko = 지도의 오류 또는 오가닉 맵으로 편집할 수 없는 항목에 대해 설명합니다.
    lt = Aprašykite žemėlapyje esančias klaidas arba dalykus, kurių negalima redaguoti naudojant "Organic Maps
    mr = नकाशामधील त्रुटी किंवा सेंद्रिय नकाशे वापरून काय संपादित केले जाऊ शकत नाही याचे वर्णन करा
    nb = Beskriv feil på kartet eller ting som ikke kan redigeres med Organic Maps
    nl = Beschrijf fouten op de kaart of dingen die niet bewerkt kunnen worden met Organic Maps
    pl = Proszę opisać błędy na mapie lub rzeczy, których nie można edytować za pomocą Organic Maps.
    pt = Descreva erros no mapa ou coisas que não podem ser editadas com o Organic Maps
    ro = Descrieți erorile de pe hartă sau lucrurile care nu pot fi editate cu Organic Maps
    ru = Опишите ошибки на карте или то, что нельзя редактировать с помощью Organic Maps
    sk = Popíšte chyby na mape alebo veci, ktoré sa nedajú upraviť pomocou aplikácie Mapy Organic
    sv = Beskriv fel på kartan eller saker som inte kan redigeras med Organic Maps
    sw = Eleza hitilafu kwenye ramani au kile ambacho hakiwezi kuhaririwa kwa kutumia Ramani za Kikaboni
    th = อธิบายข้อผิดพลาดในแผนที่หรือสิ่งที่ไม่สามารถแก้ไขได้โดยใช้แผนที่ทั่วไป
    tr = Haritadaki hataları veya Organik Haritalar ile düzenlenemeyen şeyleri tanımlayın
    uk = Опишіть помилки на карті або те, що не можна редагувати за допомогою Organic Maps
    vi = Mô tả lỗi trên bản đồ hoặc những gì không thể chỉnh sửa bằng Bản đồ không phải trả tiền
    zh-Hans = 描述地图上的错误或无法用有机地图编辑的内容
    zh-Hant = 描述地圖中的錯誤或無法使用有機地圖編輯的內容
```

Author: The shorter version is good. For the 'optional' I prefer the version after the hint in brackets, it stands out more that way.

Author: The third option "Note to OpenStreetMap volunteers (optional)" is also good, so let's proceed with that one

  [editor_about_osm]
    tags = android

>     en = Your edits are uploaded to the public <a href="https://wiki.openstreetmap.org/wiki/About_OpenStreetMap">OpenStreetMap</a> database. Please do not add personal or copyrighted information.

Reviewer: en = Your edits are uploaded to the public <a href="https://wiki.openstreetmap.org/wiki/About_OpenStreetMap">OpenStreetMap</a> database. Please do not add personal or copyrighted information.
    af = Jou wysigings word opgelaai na die publieke <a href="https://wiki.openstreetmap.org/wiki/About_OpenStreetMap">OpenStreetMap</a>-databasis. Moet asseblief nie persoonlike of kopiereginligting byvoeg nie.
    ar = يتم تحميل تعديلاتك على قاعدة البيانات العامة <a href="https://wiki.openstreetmap.org/wiki/About_OpenStreetMap">خريطة الشارع المفتوح</a>. يرجى عدم إضافة معلومات شخصية أو محمية بحقوق الطبع والنشر.
    az = Redaktələriniz ictimai <a href="https://wiki.openstreetmap.org/wiki/About_OpenStreetMap">OpenStreetMap</a> verilənlər bazasına yüklənir. Zəhmət olmasa şəxsi və ya müəllif hüquqları ilə qorunan məlumatları əlavə etməyin.
    be = Вашы праўкі загружаюцца ў публічную базу дадзеных <a href="https://wiki.openstreetmap.org/wiki/About_OpenStreetMap">OpenStreetMap</a>. Калі ласка, не дадавайце асабістую інфармацыю або інфармацыю, абароненую аўтарскім правам.
    bg = Вашите редакции се качват в публичната база данни <a href="https://wiki.openstreetmap.org/wiki/About_OpenStreetMap">OpenStreetMap</a>. Моля, не добавяйте лична информация или информация, защитена с авторски права.
    ca = Les vostres edicions es pengen a la base de dades pública <a href="https://wiki.openstreetmap.org/wiki/About_OpenStreetMap">OpenStreetMap</a>. Si us plau, no afegiu informació personal o amb drets d'autor.
    cs = Vaše úpravy se nahrají do veřejné <a href="https://wiki.openstreetmap.org/wiki/About_OpenStreetMap">databáze OpenStreetMap</a>. Nepřidávejte prosím osobní informace nebo informace chráněné autorskými právy.
    da = Dine ændringer uploades til den offentlige <a href="https://wiki.openstreetmap.org/wiki/About_OpenStreetMap">OpenStreetMap</a>-database. Tilføj venligst ikke personlige eller ophavsretligt beskyttede oplysninger.
    de = Ihre Bearbeitungen werden in die öffentliche <a href="https://wiki.openstreetmap.org/wiki/About_OpenStreetMap">OpenStreetMap</a> Datenbank hochgeladen. Bitte fügen Sie keine persönlichen oder urheberrechtlich geschützten Informationen hinzu.
    el = Οι επεξεργασίες σας μεταφορτώνονται στη δημόσια βάση δεδομένων <a href="https://wiki.openstreetmap.org/wiki/About_OpenStreetMap">OpenStreetMap</a>. Παρακαλούμε μην προσθέτετε προσωπικές πληροφορίες ή πληροφορίες που προστατεύονται από πνευματικά δικαιώματα.
    es = Sus ediciones se cargan en la base de datos pública <a href="https://wiki.openstreetmap.org/wiki/About_OpenStreetMap">OpenStreetMap</a>. Por favor, no añada información personal o protegida por derechos de autor.
    et = Teie muudatused laaditakse üles avalikku <a href="https://wiki.openstreetmap.org/wiki/About_OpenStreetMap">OpenStreetMap</a> andmebaasi. Palun ärge lisage isiklikku või autoriõigusega kaitstud teavet.
    eu = Zure aldaketak <a href="https://wiki.openstreetmap.org/wiki/About_OpenStreetMap">OpenStreetMap</a> datu-base publikora kargatzen dira. Mesedez, ez gehitu informazio pertsonalik edo copyrightdun informaziorik.
    fa = ویرایش های شما در پایگاه داده عمومی <a href="https://wiki.openstreetmap.org/wiki/About_OpenStreetMap">OpenStreetMap</a> آپلود می شود. لطفا اطلاعات شخصی یا دارای حق چاپ را اضافه نکنید.
    fi = Muokkauksesi ladataan julkiseen <a href="https://wiki.openstreetmap.org/wiki/About_OpenStreetMap">OpenStreetMap</a>-tietokantaan. Älä lisää henkilökohtaisia tai tekijänoikeudella suojattuja tietoja.
    fr = Vos modifications sont téléchargées dans la base de données publique <a href="https://wiki.openstreetmap.org/wiki/About_OpenStreetMap">OpenStreetMap</a>. Veuillez ne pas ajouter d'informations personnelles ou protégées par le droit d'auteur.
    he = העריכות שלך מועלות למסד הנתונים הציבורי של <a href="https://wiki.openstreetmap.org/wiki/About_OpenStreetMap">OpenStreetMap</a>. נא לא להוסיף מידע אישי או מוגן בזכויות יוצרים.
    hi = आपके संपादन सार्वजनिक <a href="https://wiki.openstreetmap.org/wiki/About_OpenStreetMap">OpenStreetMap</a> डेटाबेस पर अपलोड किए जाते हैं। कृपया व्यक्तिगत या कॉपीराइट जानकारी न जोड़ें।
    hu = Az Ön szerkesztései feltöltődnek a nyilvános <a href="https://wiki.openstreetmap.org/wiki/About_OpenStreetMap">OpenStreetMap</a> adatbázisba. Kérjük, ne adjon hozzá személyes vagy szerzői jogvédelem alatt álló információkat.
    id = Hasil editan Anda akan diunggah ke database <a href="https://wiki.openstreetmap.org/wiki/About_OpenStreetMap">OpenStreetMap</a> publik. Mohon untuk tidak menambahkan informasi pribadi atau informasi yang memiliki hak cipta.
    it = Le sue modifiche vengono caricate nel database pubblico <a href="https://wiki.openstreetmap.org/wiki/About_OpenStreetMap">OpenStreetMap</a>. La preghiamo di non aggiungere informazioni personali o protette da copyright.
    ja = あなたの編集は公開されている<a href="https://wiki.openstreetmap.org/wiki/About_OpenStreetMap">OpenStreetMap</a>データベースにアップロードされます。個人情報や著作権のある情報は追加しないでください。
    ko = 수정한 내용은 공개 <a href="https://wiki.openstreetmap.org/wiki/About_OpenStreetMap">OpenStreetMap</a> 데이터베이스에 업로드됩니다. 개인 정보나 저작권이 있는 정보는 추가하지 마세요.
    lt = Jūsų pakeitimai įkeliami į viešą <a href="https://wiki.openstreetmap.org/wiki/About_OpenStreetMap">OpenStreetMap</a> duomenų bazę. Prašome nedėti asmeninės ar autorių teisėmis apsaugotos informacijos.
    mr = तुमची संपादने सार्वजनिक <a href="https://wiki.openstreetmap.org/wiki/About_OpenStreetMap">OpenStreetMap</a> डेटाबेसवर अपलोड केली जातात. कृपया वैयक्तिक किंवा कॉपीराइट केलेली माहिती जोडू नका.
    nb = Dine endringer lastes opp til den offentlige <a href="https://wiki.openstreetmap.org/wiki/About_OpenStreetMap">OpenStreetMap</a>-databasen. Vennligst ikke legg til personlig eller opphavsrettsbeskyttet informasjon.
    nl = Uw bewerkingen worden geüpload naar de openbare <a href="https://wiki.openstreetmap.org/wiki/About_OpenStreetMap">OpenStreetMap</a> database. Voeg geen persoonlijke of auteursrechtelijk beschermde informatie toe.
    pl = Państwa zmiany są przesyłane do publicznej bazy danych <a href="https://wiki.openstreetmap.org/wiki/About_OpenStreetMap">OpenStreetMap</a>. Prosimy nie dodawać informacji osobistych lub chronionych prawem autorskim.
    pt = As suas edições são carregadas para a base de dados pública <a href="https://wiki.openstreetmap.org/wiki/About_OpenStreetMap">OpenStreetMap</a>. Por favor, não adicione informações pessoais ou protegidas por direitos de autor.
    pt-BR = Suas edições são carregadas no banco de dados público <a href="https://wiki.openstreetmap.org/wiki/About_OpenStreetMap">OpenStreetMap</a>. Não adicione informações pessoais ou protegidas por direitos autorais.
    ro = Edițiile dvs. sunt încărcate în baza de date publică <a href="https://wiki.openstreetmap.org/wiki/About_OpenStreetMap">OpenStreetMap</a>. Vă rugăm să nu adăugați informații personale sau protejate prin drepturi de autor.
    ru = Ваши правки будут загружены в общедоступную базу данных <a href="https://wiki.openstreetmap.org/wiki/About_OpenStreetMap">OpenStreetMap</a>. Пожалуйста, не добавляйте личную информацию или информацию, защищенную авторским правом.
    sk = Vaše úpravy sa nahrajú do verejnej databázy <a href="https://wiki.openstreetmap.org/wiki/About_OpenStreetMap">OpenStreetMap</a>. Prosím, nepridávajte osobné informácie alebo informácie chránené autorskými právami.
    sv = Dina redigeringar laddas upp till den offentliga <a href="https://wiki.openstreetmap.org/wiki/About_OpenStreetMap">OpenStreetMap</a>-databasen. Lägg inte till personlig eller upphovsrättsskyddad information.
    sw = Mabadiliko yako yanapakiwa kwenye hifadhidata ya <a href="https://wiki.openstreetmap.org/wiki/About_OpenStreetMap">OpenStreetMap</a> ya umma. Tafadhali usiongeze maelezo ya kibinafsi au yenye hakimiliki.
    th = การแก้ไขของคุณจะถูกอัปโหลดไปยังฐานข้อมูล <a href="https://wiki.openstreetmap.org/wiki/About_OpenStreetMap">OpenStreetMap</a> สาธารณะ กรุณาอย่าเพิ่มข้อมูลส่วนบุคคลหรือข้อมูลที่มีลิขสิทธิ์
    tr = Düzenlemeleriniz halka açık <a href="https://wiki.openstreetmap.org/wiki/About_OpenStreetMap">OpenStreetMap</a> veritabanına yüklenir. Lütfen kişisel veya telif hakkıyla korunan bilgiler eklemeyin.
    uk = Ваші правки завантажуються до публічної бази даних <a href="https://wiki.openstreetmap.org/wiki/About_OpenStreetMap">OpenStreetMap</a>. Будь ласка, не додавайте особисту або захищену авторським правом інформацію.
    vi = Các chỉnh sửa của bạn sẽ được tải lên cơ sở dữ liệu <a href="https://wiki.openstreetmap.org/wiki/About_OpenStreetMap">OpenStreetMap</a> công khai. Vui lòng không thêm thông tin cá nhân hoặc có bản quyền.
    zh-Hans = 您的编辑内容将上传到公共<a href="https://wiki.openstreetmap.org/wiki/About_OpenStreetMap">OpenStreetMap</a>数据库。请勿添加个人或受版权保护的信息。
    zh-Hant = 您的編輯內容將上傳至公用 <a href="https://wiki.openstreetmap.org/wiki/About_OpenStreetMap">OpenStreetMap</a> 資料庫。請不要添加個人或版權資訊。
